packages/server/package.json

```diff
@@ -98,6 +98,7 @@
     "ospath": "1.2.2",
     "p-defer": "^3.0.0",
     "p-queue": "6.1.0",
+    "performance-now": "^2.1.0",
```

Comment:
Is this used anywhere? Node has `performance.now()` :thinking:

Reply:
Good call. This was from a previous PR (I forgot to add the dependency). A lot of the benchmarking that had been done with v8-snapshots previously used `performance-now`, but I think I don't see a reason not to use `performance.now()`. Updated to use `performance.now()` here: [`c3e4634` (#23257)](https://github.com/cypress-io/cypress/pull/23257/commits/c3e46342d4b31d7bf8f5b89ccd5c512ac4e53a19)